main.cpp

```diff
@@ -1 +1,238 @@
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+#include <charconv>
+#include <algorithm>
+#include <iterator>
+#include <stdio.h>
+
+using namespace std;
+
+void print(int guesses = 0) { // print the hangman
+if (guesses >= 1) {cout << "  | ";}
```

Comment:
Again, Immersive standard is bracket and new line, even for a single line
And again, for personal project like this, it's perfectly fine